theme/acquia_lift.wizard.theme.inc

```diff
@@ -45,6 +46,9 @@ function theme_acquia_lift_card($variables) {
   }
   $html .= '</div>';
   $html .= '<div class="el-card__content">';
+  if (!empty($card_flag)) {
```

Comment:
For the purposes of this story... how about adding the transitions since we know we will have to deal with client-side adjustments to the visibility of the flag?  We can work out the other stuff as we go.